{"name": "Проверка корректной ошибки при попытки авторизации с некорректным паролем", "status": "passed", "attachments": [{"name": "stdout", "source": "df8ac744-19e9-4fc3-b286-c5d99dd12112-attachment.txt", "type": "text/plain"}], "start": 1733867478178, "stop": 1733867478496, "uuid": "7b1eba5b-2b1d-4b59-993f-f5c35251d4aa", "historyId": "9699b600ff3621aeb7cf0246a5fd1dd0", "testCaseId": "9699b600ff3621aeb7cf0246a5fd1dd0", "fullName": "tests.test_login user.TestLoginUser#test_login_with_incorrect_password", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "Karinas-MacBook-Pro.local"}, {"name": "thread", "value": "8223-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login user"}]}